src/power_grid_model/core/power_grid_model.py

```diff
@@ -272,55 +277,57 @@ def calculate_power_flow(
         continue_on_batch_error: bool = False,
     ) -> Dict[str, np.ndarray]:
         """
-        Calculate power flow once with the current model attributes.
-        Or calculate in batch with the given update dataset in batch
+        Calculate power flow once with the current model attributes. 
```

Comment:
```suggestion
        Calculate power flow once with the current model attributes.
```

Reply:
The original docstring in the main branch contains two lines.

"Calculate state estimation once with the current model attributes.
Or calculate in batch with the given update dataset in batch"